webpack.config.js

```diff
@@ -90,17 +115,16 @@ const webpackConfig = {
 		fs: 'empty'
 	},
 	plugins: [
+		new DashboardPlugin(),
 		new webpack.DefinePlugin( {
 			'process.env': {
 				NODE_ENV: JSON.stringify( bundleEnv )
 			},
 			'PROJECT_NAME': JSON.stringify( config( 'project' ) )
 		} ),
-		new WebpackStableBuildPlugin( {
-			seed: 0
-		} ),
 		new webpack.IgnorePlugin( /^props$/ ),
-		new CopyWebpackPlugin( [ { from: 'node_modules/flag-icon-css/flags/4x3', to: 'images/flags' } ] )
+		new CopyWebpackPlugin( [ { from: 'node_modules/flag-icon-css/flags/4x3', to: 'images/flags' } ] ),
+		new webpack.HashedModuleIdsPlugin(),
```

Comment:
I don't think so.  I removed the two different NamedHashes/NamedChunks before because I thought they were breaking the runtime.  I'm adding them back now that I've isolated it to something else